prysm/chain.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chain.schema.json",
+  "chain_name": "prysm",
+  "chain_type": "cosmos",
+  "status": "live",
+  "website": "https://www.lightlabs.dev",
+  "network_type": "testnet",
+  "pretty_name": "Prysm",
+  "chain_id": "prysm-devnet-1",
```

Comment:
Is this for the devnet? If so, this is already registered: https://github.com/cosmos/chain-registry/tree/master/testnets/prysmdevnet